robotbase/src/main/java/frc/robot/subsystems/Pivot.java

```diff
@@ -52,116 +53,75 @@ private void configure() {
     m_pivotPIDController.setFeedbackDevice(m_absoluteEncoder);
   }
 
-  public void setManualRotation(double rotation) {
-    Robot.state.setPivotState(PivotState.CLOSED_LOOP);
-    setPivotRotation(rotation);
+  public boolean isZeroed() {
+    return m_isZeroed;
   }
 
-  public void setPivotRotation(double rotation) {
-    m_targetRotation = rotation;
-    m_pivotPIDController.setReference(m_targetRotation, ControlType.kPosition);
+  public boolean isStopped() {
+    return Double.isNaN(m_targetAngle) && Double.isNaN(m_axisSpeed);
   }
 
-  public void setPivotAxisSpeed(double axisSpeed) {
-    Robot.state.setPivotState(PivotState.NONE);
+  public boolean isSettingAngle() {
+    return !Double.isNaN(m_targetAngle);
+  }
+
+  public boolean isRunningAxisSpeed() {
+    return !Double.isNaN(m_axisSpeed);
```

Comment:
I'm starting to wonder if we event need m_axisSpeed.  Our commands are now controlling the state of the subsystem, and the motors are always stopped at the end of them, so we don't really need to track anything in the subsystem it seems like. The only thing I see us using m_targetAngle for is just to check if the pivot at the setpoint.